.github/workflows/presubmit.yaml

```diff
@@ -82,34 +80,6 @@ jobs:
       with:
         name: debs
         path: debs.tar
-  build-cvd:
-    runs-on: ubuntu-22.04-4core
-    container:
-      image: debian@sha256:9258a75a7e4323c9e5562b361effc84ee747920116d8adfc98a465a5cdc9150e # debian:bookworm-20250407 (amd64)
-    steps:
-    - name: Common bazel setup
-      uses: bazel-contrib/setup-bazel@f3f50ea6791b9b0f4c4eeabba4507422426462f5 #aka 0.9.1
-      with:
-        bazelisk-cache: true # Avoid downloading Bazel every time.
-        disk-cache: ${{ github.workflow }} # Store build cache per workflow.
-        repository-cache: true # Share repository cache between workflows.
-    - name: Checkout repository
-      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 # aka v2
-    - name: Setup apt
-      run: apt update -y && apt upgrade -y
-    - name: Install dependencies
-      run: apt install -y git clang libcurl4-openssl-dev clang-tidy libtinfo5 xxd cmake liblzma-dev libvpx-dev libopus-dev libaom-dev autoconf libtool
-    - name: Install bazel
-      run: bash tools/buildutils/installbazel.sh
-    - name: Build cvd
-      run: cd base/cvd && bazel build --verbose_failures ...
-    - name: Test cvd
-      run: cd base/cvd && bazel test --sandbox_writable_path=$HOME --test_output=errors ...
```

Comment:
Is `bazel test ...` being run in any other workflow right now?

Reply:
Good catch! Rephrased the job as run-cvd-unit-tests running only the cvd unit tests, no need to build cvd here.

Reply:
IIUC in practice there won't be a huge time saving to remove the `bazel bulid '...'` step first since `bazel test '...'` will include most of that work.

Reply:
I think the problem is that the `debuild` environment throws away the bazel cache, wasting the work of the first runner. [b/410052922](http://b/410052922) proposes using `pkg_deb` instead to address this issue, but that will be more work.

Reply:
I see, thanks.